### PR TITLE
LP-2581 Add is education experience completed flag in context

### PR DESCRIPTION
**Pull request by zeshanarifios**
[LP-2581](https://philanthropyu.atlassian.net/browse/LP-2581)

### CODE REVIEW

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1351/workflows/workflows_id/jobs/1351/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (50.0%): Missing lines 74
openedx/adg/common/lib/mandrill_client/helpers.py (100%)
openedx/adg/common/lib/mandrill_client/tests/test_helpers.py (100%)
openedx/adg/lms/applications/models.py (92.3%): Missing lines 240
openedx/adg/lms/student/helpers.py (100%)
openedx/adg/lms/student/tests/test_helpers.py (100%)
-------------
Total:   35 lines
Missing: 2 lines
Coverage: 94%
-------------
```

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1355/workflows/workflows_id/jobs/1355/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
-------------
Total:   19 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/lms/applications/models.py [hunk 1]
```diff
@@ -208,15 +208,43 @@ def prereq_course_scores(self):
...
         """
-        return not WorkExperience.objects.filter(user_application=self).exists()
+        return not self.has_work_experience
+
+    @property
+    def is_education_completed(self):
+        """
+        Check if any education is associated with the user application.
+        """
+        return Education.objects.filter(user_application=self).exists()
```
Comment (asadali145) [inline openedx/adg/lms/applications/models.py:233]:
```suggestion
        return self.applications_educations.exists()
```

File: openedx/adg/lms/applications/models.py [hunk 2]
```diff
@@ -208,15 +208,43 @@ def prereq_course_scores(self):
...
+
+    @property
+    def is_work_experience_completed(self):
+        """
+        Check if any work experience is associated with the user application.
+        """
```
Comment (asadali145) [inline openedx/adg/lms/applications/models.py:239]: Can we update the docstring to differentiate it from `has_work_experience`.

File: openedx/adg/lms/applications/views.py [hunk 3]
```diff
@@ -313,7 +313,8 @@ def get_context_data(self, **kwargs):
             'month_choices': month_choices(),
             'year_choices': year_choices(),
             'user_application_id': user_application.id,
-            'is_work_experience_not_applicable': user_application.is_work_experience_not_applicable
+            'is_work_experience_not_applicable': user_application.is_work_experience_not_applicable,
+            'is_education_experience_completed': user_application.is_education_experience_completed
```
Comment (asadali145) [inline openedx/adg/lms/applications/views.py:317]: Can we add a trailing comma to avoid 2 changes in git diff for future?

**Review by asadali145 — COMMENTED** [review]
A few small suggestions

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1356/workflows/workflows_id/jobs/1356/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
-------------
Total:   19 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by asadali145 — APPROVED** [review]
LGTM👍🏻

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1357/workflows/workflows_id/jobs/1357/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
-------------
Total:   19 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by annas-usman — APPROVED** [review]

**Review by MunamMubashir-arbisoft — APPROVED** [review]
LGTM!

**Review by ahmed-faraz46 — APPROVED** [review]
Looks great!

**Pull request merged**